src/app/components/pages/settings/blockchain/blockchain.component.ts

```diff
@@ -9,13 +10,24 @@ import { BlockchainService } from '../../../../services/blockchain.service';
 export class BlockchainComponent implements OnInit {
   block: any;
   coinSupply: any;
+  isLoading = false;
```

Comment:
This is not really necessary, checking `!(block && coinSupply)` is enough, similar to how it is done in `HistoryComponent`.

Reply:
Fixed